tests/network/network_simplification/test_network_simplification_factory.py

```diff
@@ -0,0 +1,58 @@
+from typing import Iterator
+
+import networkx as nx
+import pytest
+
+from ra2ce.network.network_simplification.network_simplification_factory import (
+    NetworkSimplificationFactory,
+)
+
+
+class TestNetworkSimplificationFactory:
+    @pytest.fixture(name="network_simplification_factory")
+    def _get_network_simplification_factory(
+        self,
+    ) -> Iterator[NetworkSimplificationFactory]:
+        yield NetworkSimplificationFactory(graph_complex=None, attributes_to_exclude=[])
+
+    def test_validate_fixture_init(
```

Comment:
Do you consider it useful/helpful to test the fixture instead of having a separate test on the initalization of a class?

Reply:
Yes because it is directly testing that we can initialize the class with `None` and `[]` as arguments. This fixture is also used in the other tests.